packages/makeup-exit-emitter/test/index.js

```diff
@@ -21,33 +23,58 @@ describe('given an element with focus', function() {
         testElSibling = document.querySelector('#test-element-sibling');
         ExitEmitter.addFocusExit(testEl);
         testEl.addEventListener('onFocusExit', onFocusExit);
```

Comment:
@ianmcburnie , I guess this should be 
`testEl.addEventListener('focusExit', onFocusExit);` thats why failing most of the tests..